src/libs/ReportActionComposeFocusManager.js

```diff
@@ -30,8 +32,20 @@ function clear() {
     focusCallback = null;
 }
 
+/**
+ * Exposes the current focus state of the report action composer.
+ *
+ */
+function isFocused() {
+    if (composerRef.current) {
+        composerRef.current.isFocused();
+    }
+}
```

Comment:
I've noticed this function does not `return` a value

Reply:
Oh what a mess. . PR incoming.